pennylane_lightning/src/simulator/Measures.hpp

```diff
@@ -30,10 +30,11 @@
 
 #include "Kokkos_Sparse.hpp"
 #include "LinearAlgebra.hpp"
+#include "Observables.hpp"
 #include "StateVectorManagedCPU.hpp"
 #include "StateVectorRawCPU.hpp"
 
-namespace Pennylane {
+namespace Pennylane::Simulators {
```

Comment:
I just used it as it is inside `simulator` directory. Not using sub-namespace may be better?

Reply:
I wonder if we should move this to a similar structure as PennyLane, rather than keep everything in the `simulator` dir. We don't need this for now, but I think long term the `Simulators` namespace may not be the best option here for these.

Though, if it helps with standardisation we can keep the namespace for now until we move it to a better dir later. That work?

Reply:
:+1: